jetty-websocket/websocket-jetty-server/src/main/java/org/eclipse/jetty/websocket/server/config/JettyWebSocketServletContainerInitializer.java

```diff
@@ -51,17 +52,15 @@ public static void configure(ServletContextHandler context, Configurator configu
         if (!context.isStopped())
             throw new IllegalStateException("configure should be called before starting");
 
-        context.addEventListener(
-            ContainerInitializer
-                .asContextListener(new JettyWebSocketServletContainerInitializer())
-                .afterStartup((servletContext) ->
+        context.addServletContainerInitializer(new JettyWebSocketServletContainerInitializer()
+            .afterStartup((servletContext) ->
```

Comment:
Same comment as for the javax version.
The lambda should be inlined in `onStartup()`.